src/scf/scf_callbacks.jl

```diff
@@ -65,7 +68,10 @@ Flag convergence by using the L2Norm of the change between
 input density and unpreconditioned output density (ρout)
 """
 function ScfConvergenceDensity(tolerance)
-    info -> norm(info.ρout.fourier - info.ρin.fourier) < tolerance
+    function f(info)
+        norm(info.ρout - info.ρin) * sqrt(info.basis.dvol) < tolerance
+    end
+    f
```

Comment:
Really? Come on that's four lines for something that is super clear in my opinion in one line.

Reply:
I didn't find it very clear but OK, I'll just add parens